### ISSUE TITLE
fcos training get nan loss

### ISSUE THREAD

**Issue by gneworld** [open-mmlab/mmdetection#4377]
hi, guys
thanks for your great job. When I try to make a test about fcos with "python tools/train.py configs/fcos/fcos_center-normbbox-centeronreg-giou_r50_caffe_fpn_gn-head_4x4_1x_coco.py --work-dir output/", I got a nan loss only after  250 iterations, like this 

2020-12-31 21:47:04,494 - mmdet - INFO - Epoch [1][50/29317]    lr: 3.987e-03, eta: 2 days, 4:20:34, time: 0.536, data_time: 0.154, memory: 5679, loss_cls: 1.2290, loss_bbox: 0.7894, loss_centerness: 0.6860, loss: 2.7043
2020-12-31 21:47:26,052 - mmdet - INFO - Epoch [1][100/29317]   lr: 4.653e-03, eta: 1 day, 23:12:41, time: 0.431, data_time: 0.054, memory: 5708, loss_cls: 1.4720, loss_bbox: 0.5644, loss_centerness: 0.6886, loss: 2.7251
2020-12-31 21:47:46,457 - mmdet - INFO - Epoch [1][150/29317]   lr: 5.320e-03, eta: 1 day, 20:45:23, time: 0.408, data_time: 0.015, memory: 5717, loss_cls: 1.5323, loss_bbox: 0.5674, loss_centerness: 0.6745, loss: 2.7742
2020-12-31 21:48:09,300 - mmdet - INFO - Epoch [1][200/29317]   lr: 5.987e-03, eta: 1 day, 20:43:02, time: 0.457, data_time: 0.079, memory: 5717, loss_cls: 1.5484, loss_bbox: 0.5734, loss_centerness: 0.6843, loss: 2.8061
2020-12-31 21:48:28,971 - mmdet - INFO - Epoch [1][250/29317]   lr: 6.653e-03, eta: 1 day, 19:27:13, time: 0.393, data_time: 0.015, memory: 5717, loss_cls: 13794.6467, loss_bbox: nan, loss_centerness: nan, loss: nan

I didn't modify anything with code except the data path, so what's wrong with this or am I missing  anything?  thanks much

**Comment by ZwwWayne** [comment]
The learning rate seems to be not changed according to your smaller batch size. You may also check the [solution in FAQ](https://mmdetection.readthedocs.io/en/latest/faq.html#training).
